### ISSUE TITLE
[Bug]: Escalated appeals should have a better wording in version history

### ISSUE THREAD

**Issue by ioanarusiczki** [mozilla/addons#14809]
### What happened?

From https://github.com/mozilla/addons/issues/2010#issuecomment-2134501006

1. Report an extension or theme to go into Listings or Themes Cinder queues
2. Resolve the report from Cinder 
3. Reporters or authors send an appeal
4. Escalate to AMO the appeal from Cinder 
5. Go to rev tools Manual Review Queue  and observe the version history on the add-on's review page

![escalated appeal from cinder](https://github.com/mozilla/addons/assets/33448286/2ffb3f87-d623-4e0e-a1f7-c6791fc66d61)

### What did you expect to happen?

Maybe it should be "Escalated appeal via Cinder"

### Is there an existing issue for this?

- [X] I have searched the existing issues

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/AMOENG-795)

**Comment by diox** [comment]
Andreas, what's your opinion on this?

**Comment by wagnerand** [comment]
I agree that the entity that was escalated should be accurate. In this case an appeal was escalated, so the reviewer tools should say so accordingly.

This might be a separate issue, but we should also make sure that in this case all information about this report is shown to the reviewer, so they can understand and assess the appeal. This includes:
* the abuse report itself, including the category and additional details
* the decision of the moderator, including any details provided
* the appeal details provided by the reporter

It's unclear from the screenshot if that information is already shown somewhere.

**Comment by ioanarusiczki** [comment]
This is the details as available now:

![Information available in the DSA section](https://github.com/user-attachments/assets/52c458af-ed3a-467b-b9a5-0e0b04af30a3)

**Comment by ioanarusiczki** [comment]
Lgtm on AMO dev

![Image](https://github.com/user-attachments/assets/3050a076-92cf-4bb4-a744-d9fb879abe5a)
